lms/static/sass/xblocks/assessment.scss

```diff
@@ -0,0 +1,104 @@
+@import 'vars/variables';
+@import 'mixins/mixins';
+@import 'core/reset';
+@import 'core/core';
+@import 'form/checkbox';
+@import 'form/radiobutton';
+@import 'common/buttons';
+
+
+
+.new-theme
+{
+    [data-block-type="step-builder"]
```

Comment:
@shahidrafique Please Do code formatting on all SCSS files.

Reply:
this is individual SCSS for xBlock.